docs/content/latest/yugabyte-platform/security/security-checklist-yp.md

```diff
@@ -23,7 +23,7 @@ Read more about enabling [Encryption in transit](../enable-encryption-at-rest) i
 ## Enable encryption at rest
 
 [Encryption at rest](https://en.wikipedia.org/wiki/Data_at_rest#Encryption) ensures that data at rest, stored on disk, is protected. You can configure YugabyteDB with a user-generated symmetric key to perform cluster-wide encryption.
-Read more about enabling Encryption at rest in YugabyteDB.
+Read more about [enabling Encryption at rest](../enable-encryption-at-rest) in YugabyteDB.
```

Comment:
This link is right, but the one on line 21 is incorrect! (That one should point to `../enable-encryption-in-transit`)

Reply:
I think my latest commit fixes it.